colour/utilities/array.py

```diff
@@ -1788,6 +1792,142 @@ def from_range_int(
     return a
 
 
+_NDARRAY_COPY_ENABLED: bool = True
```

Comment:
I wonder if there is something we should do (long term, not this PR) to manage the number of globals Colour creates. A lot of the caches are also globals. Users might not want to see these globals if they are trying to debug client applications, or at least it could be clearer that they are colour-"globals"

Reply:
This is a good point yeah, it should be easy to do!